Author: Doug Goldstein <cardoe@gentoo.org>
Date: Thr Sep 13 10:08:24 2007 +0400
 remove shadow depend on libattr and make it user config'd

  removes the shadow depend on libattr and makes it user configurable.
  leaves the default to yes as is currently in gedit, however it fails
  if libattr is not found so that it's not a shadow depend.

--- gedit-2.18.2/configure.ac.old	2007-06-24 12:06:30.000000000 -0400
+++ gedit-2.18.2/configure.ac	2007-09-13 10:21:12.000000000 -0400
@@ -138,8 +138,15 @@
 dnl ================================================================
 dnl libattr checks
 dnl ================================================================
-
-AC_CHECK_LIB(attr, attr_copy_fd)
+AC_ARG_WITH([attr],
+	AS_HELP_STRING([--without-attr],[Don't use libattr support]),
+	[],
+	[with_attr=yes])
+
+if test "x$with_attr" = "xyes"; then
+	AC_CHECK_LIB([attr], [attr_copy_fd], [],
+		AC_MSG_ERROR([*** attr library (libattr) not found]))
+fi
 
 dnl ================================================================
 dnl Start of pkg-config checks
